### PR TITLE
Content review of code of conduct

### PR DESCRIPTION
**Pull request by lauraykerr**
Minor language edits for plain language, consistency, and clarity, plus onc replaced broken link

# PULL REQUEST

## Related Issue

Belated content review of PR [#3783](https://github.com/CDCgov/prime-simplereport/pull/3783)

## Additional Information

- Plain language and clarity edits, such as active voice and consistence use of "you" (instead of "I"). 
- Replaced link for Clearance of Information Products Disseminated Outside CDC for Public Use, since existing link led to 404. 

## Checklist for Primary Reviewer

- [ ] Edits preserve the intended meaning of the content

### CODE REVIEW

**Comment by lauraykerr** [comment]
Note: Largely left "Background" section as is, since it seems to be a legal inheritance from the CDC. Or do you think there's more leeway to edit? @rin-skylight

File: code-of-conduct.md [hunk 1]
```diff
@@ -1,103 +1,108 @@
...
 to create solutions for the challenges we face.
 
-This is our code of conduct (adapted from [18F's Code of Conduct](https://github.com/18F/code-of-conduct)).
+This is our code of conduct (adapted from [18F's code of conduct](https://github.com/18F/code-of-conduct)).
 We follow all Equal Employment Opportunity laws and we expect everyone we work
-with to adhere to the [GSA Anti-harrasment Policy](http://www.gsa.gov/portal/directive/d0/content/512516),
-even if they do not work for the Centers for Disease Control and Prevention or
-GSA. We expect every user to follow this code of conduct and the laws and
-policies mentioned above.
+with to adhere to the [US General Services Administation (GSA) anti-harrasment policy](http://www.gsa.gov/portal/directive/d0/content/512516),
```
Comment (rin-skylight) [inline code-of-conduct.md:11]: This wasn't part of your changeset, but I think `harassment` is misspelled, here.

**Review by rin-skylight — APPROVED** [review]
I have no objections to any of these changes! I'm all for cleaning up and modernizing the language. Thank you for going over this!

I've made a single suggestion, but it is non-blocking. It was something I should have caught on my initial submission, and am more than happy to make a separate PR for it.

**Review by emmastephenson — APPROVED** [review]
LGTM, thanks for following up on this Laura!

**Pull request merged**